python/lsst/sdm/schemas/hsc.yaml

```diff
@@ -851,6 +841,81 @@ tables:
     datatype: float
     description: HSM moments. Measured on g-band.
     fits:tunit: pixel**2
+  - name: g_moments_uu
+    "@id": "#Object.g_moments_uu"
+    datatype: float
+    description: HSM moments in ra,dec coordinates. Measured on g-band.
+    fits:tunit: arcsec**2
+  - name: g_moments_vv
+    "@id": "#Object.g_moments_vv"
+    datatype: float
+    description: HSM moments in ra,dec coordinates. Measured on g-band.
+    fits:tunit: arcsec**2
+  - name: g_moments_uv
+    "@id": "#Object.g_moments_uv"
+    datatype: float
+    description: HSM moments in ra,dec coordinates. Measured on g-band.
+    fits:tunit: arcsec**2
+  - name: g_moments_flag
+    "@id": "#Object.g_moments_flag"
+    datatype: boolean
+    description: General failure flag for moments measurement.
+    fits:tunit:
+  - name: g_moments_psf_uu
+    "@id": "#Object.g_moments_psf_uu"
+    datatype: float
+    description: PSF moments in ra,dec coordinates. Measured on g-band.
```

Comment:
I'd similarly replace these docstrings with:

> Adaptive Gaussian-weighted second moments of the PSF model at the position of the object, in a local gnomonic projection, with `u` aligned with RA and `v` aligned with declination. Measured on g-band.

We should ask @arunkannawadi or @jmeyers314 how to qualify that for the debiased and "round" moments in a succinct way; I _think_ I remember how those work but am not certain enough to propose text.